assignment-2/power.md

```diff
@@ -1,3 +1,127 @@
+
+According to another report of the office of inspector general [3] CMS did not alway manage and oversee the contractor performance. The main problem in this case is the lack of control on the contract deliverables.
+
+######Obama administration
+Also the Obama administration, did not make this project easy, it kept modifying regulations and policies of the Affordable Care Act [1], which inherently meant that the scope of the HealthCare.gov website kept changing too. Which probably has caused some hardship with the changes trickling down to all the contractors, but there is a lack of documentation on how the changes were documented and implemented.
+
+
+#######Culture: Alienative, Calculative vs. Moral involvement
+Etzioni [6] [7] developed the compliance theory. He classifies organizations by the type of power they use to direct the behavior of their members and the type of involvement of the participants. He identified three types of organizational power: coercive, utilitarian, and normative, and relates these respectively to three types of involvement: alienative, calculative, and moral.  
+
+
+(still need to written coupling between Etzioni theory and culture in Washington DC)
+
+According to Robert Kennedy [8] a Google employee that worked in is free time in Washington DC at the HealthCare.gov project people were afraid of making mistakes, because they could be fired for it. This way people did not raise any mistakes that they made and therefore the problems that result from these mistakes couldn’t be easily resolved.
+
+It seems that this culture of fear resulted that on vital reports were not reacted on the right way. For example on April 4, 2013 a 15-page document by McKinsey & Co. warns among others the CMS administrator that there was insufficient end-to-end testing and advises that a limited initial launch of the website would be ideal. This reports surfaced only half November 2013. One and a half month after the lauch. The advise in this report was not followed, because there were only two weeks of end-to-end testing before the launch took place for all the users and there was no limited initial lauch. Also other reports and warnings were discarded like that the website as of September 30 could handle only about 1100 users at a time, even though officials have said it should have been able to accomodate perhaps as many as 60,000 users. It seems that people were afraid to be fired or were afraid that the Republicans would use such reports to try to delay or cancel the whole project and with that the whole Obamacare bill.
+
+
+Robert Kennedy [8] also stated that he worked met with an employee of one of the site contractors who was in charge of software “security.” When he asked her which penetration threats she had prepared for, she responded that she didn’t think about such things: she was in charge of “risk assessment.”
```

Comment:
- 'worked met' is now 'met'
- I will elaborate that. I think the link with power is that people try to take as less responsibility because the are afraid to get punished. Not very empowering.